Nodejs/Product/Nodejs/Debugger/NodeDebugger.cs

```diff
@@ -1218,11 +1272,35 @@ public NodeDebugger(Uri debuggerEndpointUri, int id)
             javaScriptFileName = FileNameMapper.GetLocalFileName(javaScriptFileName);
 
             // Try to get mapping for JS file
-            if(stackFrame != null) {
+            if (stackFrame != null) {
                 line = stackFrame.Line;
                 column = stackFrame.Column;
             }
-            string originalFileName = SourceMapper.GetOriginalFileName(javaScriptFileName, line, column);
+
+            SourceMapSourceInfo sourceInfo = null;
+            string name;
+
+            string sourceMapFilename = FindSourceMapFile(javaScriptFileName);
+
+            DecodedSourceMap decodedSourceMap;
+
+            if (!string.IsNullOrEmpty(sourceMapFilename)) {
+
+                if (DecodedSourceMaps.ContainsKey(sourceMapFilename)) {
+                    decodedSourceMap = DecodedSourceMaps[sourceMapFilename];
+                } else {
+                    decodedSourceMap = SourceMapReader.LoadSourceMap(javaScriptFileName, sourceMapFilename);
+                    DecodedSourceMaps.Add(sourceMapFilename, decodedSourceMap);
+                    _sourceMapFiles.Add(javaScriptFileName, sourceMapFilename);
+                }
+
+                decodedSourceMap.MapJsSourcePosition(new DkmTextSpan((int)line + 1, (int)line + 1, (int)column + 1, (int)column + 1), out sourceInfo, out name);

```

Comment:
define name above this file since you don't actually use it for anything other than the output arg